### PR TITLE
setting up data marts

### PR DESCRIPTION
**Pull request by AleckVictor**
- Creating data marts folder
- Creating data mart views models

### CODE REVIEW

File: models/mart/historical_mrr.sql [hunk 1]
```diff
@@ -0,0 +1,99 @@
+with product as (
+select
+	id,
+	name,
+	created
+from
+	{{source('dbt_stripe_account_src', 'product')}}
+where
+	active = true),
+ device as (
```
Comment (fmluizao) [inline models/mart/historical_mrr.sql:10]:
```suggestion
	active = true
),
device as (
```

File: models/mart/historical_mrr.sql [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+	created,
+	row_number() over (partition by json_extract_path_text(custom_field_data,
+	'subscription_item_id')
+order by
+	json_extract_path_text(custom_field_data,
+	'subscription_item_id'),
+	created desc) rn
+from
+	{{source('ft_netbox_public', 'dcim_device')}}),
+	item_info as (
```
Comment (fmluizao) [inline models/mart/historical_mrr.sql:24]:
```suggestion
	{{source('ft_netbox_public', 'dcim_device')}}
),
item_info as (
```

File: models/mart/historical_mrr.sql [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+		when lower(nickname) ilike '%japan%' then 'Japan'
+		when lower(nickname) ilike '%united-states%' then 'United States'
+		when lower(nickname) ilike '%united states%' then 'United States'
+		when lower(nickname) ilike '%argentina%' then 'Argentina'
+		when lower(nickname) ilike '%chile%' then 'Chile'
+		when lower(nickname) ilike '%mexico%' then 'Mexico'
+		when lower(nickname) ilike '%new york%' then 'United States'
+		when lower(nickname) ilike '%united-kingdom%' then 'United Kingdom'
+	end as price_local
+from {{source('dbt_stripe_account_src', 'price')}}),
```
Comment (fmluizao) [inline models/mart/historical_mrr.sql:56]:
```suggestion
from {{source('dbt_stripe_account_src', 'price')}}
),
```

File: models/mart/historical_mrr.sql [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+left join {{source('dbt_stripe_account_src', 'customer')}} c on
+	silim.customer_id = c.id and silim.stripe_account = c.stripe_account 
+left join {{source('dbt_stripe_account_src', 'plan')}} plan on
+	plan.id = silim.plan_id and silim.stripe_account = plan.stripe_account 
+left join product p on
+	plan.product_id = p.id
+left join {{source('dbt_stripe_account_src', 'price')}} price on plan.id = price.id
+left join price_location pl on pl.id = silim.plan_id
+order by
+	invoice_date desc)
```
Comment (fmluizao) [inline models/mart/historical_mrr.sql:86]:
```suggestion
	invoice_date desc
)
```

File: models/mart/historical_mrr.sql [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+		when lower(nickname) ilike '%mexico%' then 'Mexico'
+		when lower(nickname) ilike '%new york%' then 'United States'
+		when lower(nickname) ilike '%united-kingdom%' then 'United Kingdom'
+	end as price_local
+from {{source('dbt_stripe_account_src', 'price')}}),
+mrr as (
+select 
+	silim.invoice_id,
+	silim.customer_id,
+	c."name",
```
Comment (fmluizao) [inline models/mart/historical_mrr.sql:61]: Are quotes required here? :thinking:

**Review by fmluizao — APPROVED** [review]

File: models/mart/historical_mrr.sql (hunk 5 above)
Comment (AleckVictor) [reply models/mart/historical_mrr.sql:64]: Yes... So we can identify name as a column from the table

**Pull request merged**